ros/processor/insights_engine_result_consumer.py

```diff
@@ -53,15 +53,13 @@ def run(self):
         for msg in iter(self):
             if msg.error():
                 print(msg.error())
```

Comment:
@kgaikwad Just noticed one thing. Is this print statement needed here or should we use `log`?

Reply:
@r14chandra  - Fixed! Replaced with log statement. Created [bug](https://issues.redhat.com/browse/RHIROS-456) based on my observation.